kong/plugins/zipkin/opentracing.lua

```diff
@@ -19,6 +19,22 @@ local OpenTracingHandler = {
 
 local tracer_cache = setmetatable({}, {__mode = "k"})
 
+local function tag_with_service_and_route(ctx, span)
+  if ctx.service and ctx.service.id then
+    span:set_tag("kong.service", ctx.service.id)
+    if ctx.route and ctx.route.id then
+      span:set_tag("kong.route", ctx.route.id)
+    end
+    if type(ctx.service.name) == "string" then
+      span:set_tag("peer.service", ctx.service.name)
+    else
+      span:set_tag("peer.service", ctx.service.id)
+    end
+  elseif ctx.api and ctx.api.id then
+    span:set_tag("kong.api", ctx.api.id)
+  end
+end
```

Comment:
Should we use `pdk`, or is that too big of a change considering that code `predates`(?) PDK?

e.g.
```lua
local service = kong.router.get_service()
local route = kong.router.get_route()
```

Reply:
Yes, done!